var.mk

```diff
@@ -1,5 +1,10 @@
 # to be included by Makefile
 
+STACK_EXISTS := $(shell command -v stack 2> /dev/null)
+ifndef STACK_EXISTS
+    $(error "stack is not available please install haskell-stack")
+endif
+
```

Comment:
Close to: When you wanna use stack as currently done in the Makefile, you need to make sure, that it is actually there. If it is not there, don't use it - and this can be done by defining the related vars as shown above.